aldryn_newsblog/templates/aldryn_newsblog/includes/meta.html

```diff
@@ -0,0 +1,51 @@
+{% load meta_extra %}
+{% if meta %}
+    {% autoescape off %}
+	{% if meta.description %}{% meta 'description' meta.description %}{% endif %}
```

Comment:
Though we should find then a smarter solution by auto generating it. We need to keep it DRY